python/runner.py

```diff
@@ -37,12 +37,12 @@
 
     object_bargraph.put(Body=bargraph)
 
-    object_rolling_average = s3.Object(
+    object_density_estimation = s3.Object(
         bucket_name=os.getenv("BUCKETEER_BUCKET_NAME"),
         key=f"{config.facility}_rolling_average",
```

Comment:
We could rename to ` key=f"{config.facility}_density_estimation",`
and I'll update the server logic